### PR TITLE
[new release] bechamel (5 packages) (0.5.0)

### PR DESCRIPTION
**Pull request by dinosaure**
Yet Another Benchmark in OCaml

- Project page: <a href="https://github.com/mirage/bechamel">https://github.com/mirage/bechamel</a>
- Documentation: <a href="https://mirage.github.io/bechamel/">https://mirage.github.io/bechamel/</a>

##### CHANGES:

- Fix unsafe access causing SIGSEGV (@edwintorok, mirage/bechamel#43)
- Fix compulation of the confidence indicator (@edwintorok, @lindig, @dinosaure, mirage/bechamel#45)
- Upgrade monotonic clock detection (@dra27, mirage/bechamel#44)
- Fix double-free when we use resources (@edwintorok, @dinosaure, mirage/bechamel#47)

### CODE REVIEW

File: packages/mperf/mperf.0.5.0/opam [hunk 1]
```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer:   [ "Romain Calascibetta <romain.calascibetta@gmail.com>" ]
+authors:      [ "Vincent Bernardoff <vincent.bernardoff@ocamlpro.com>" "Pierre Chambart <pierre.chambart@ocamlpro.com>" ]
+homepage:     "http://github.com/dinosaure/bechamel"
+dev-repo:     "git://github.com/dinosaure/bechamel"
+bug-reports:  "http://github.com/dinosaure/bechamel/issues"
+license:      "GPL-2.0"
```
Comment (mseri) [inline packages/mperf/mperf.0.5.0/opam:7]: Is this correct? The repo only mentions MIT. If it is correct is it `GPL-2.0-only` or the `or-later` version?

Comment (dinosaure) [reply packages/mperf/mperf.0.5.0/opam:7]: Yes, I need to check the history because `mperf` was made by @chambart initially.

Comment (avsm) [reply packages/mperf/mperf.0.5.0/opam:7]: Waiting on this to be resolved before merging.

**Comment by mseri** [comment]
Thanks

**Pull request merged**